docs/tutor.rst

```diff
@@ -87,3 +87,23 @@ Third-party contributions to Tutor and its plugins are more than welcome! Just m
 Where "TYPE" is either "Bugfix", "Improvement", "Feature" or "Security". You should add an explosion emoji ("💥") before "[TYPE]" if you are making a breaking change.
 
 Happy hacking! ☘️
+
+Becoming a Tutor Maintainer
+---------------------------
+
+Is Tutor a core part of your infrastructure? Would you like to contribute to the project roadmap? Then you should consider becoming a Tutor maintainer. Tutor maintainers are individuals and companies who help make decisions on the future of the prject. Their goal is to support the Tutor community and to make the software as best as possible.
+
+More precisely, project maintainers are responsible for one or more of the following tasks:
+
+- Interaction with the community on the `Tutor forums <discuss.overhang.io/>`__: maintainers are there to answer questions, help users troubleshoot issues, announce new features and changes.
+- Address Github issues by triaging, commenting and, when possible, proposing a fix for them.
+- Review Github pull requests, both from external contributors and other maintainers.
+
+As a Tutor maintainer, you should expect that your role will require about 4-8 hours of work per month. If maintainers work more than that, then it means that more maintainers should be brought on board. If you find that you cannot sustain this pace of work for an extended period of time (~1 month), then we ask that you clearly signal your new situation by removing yourself from the maintainers group. This will not impede your ability to join the group again at a later time.
```

Comment:
@regisb How much time do you estimate that you spend doing this today? To get an idea of the amount of work there is overall, currently.

Reply:
- Reviewing PRs: 1-2 hours/month
- Answering questions on the forums: 10 hours/months
- Investigating bug reports from the forums and producing bug fixes: 20 h/month
- Adding new features, creating new plugins: a lot more.